src/shared/localizer.cpp

```diff
@@ -242,6 +243,45 @@ QMap<QString, double> Localizer::loadTranslationCompleteness(
   return result;
 }
 
+// static
+QString Localizer::formatDate(const QDateTime& nowDateTime,
+                                   const QDateTime& messageDateTime) {
+  qint64 diff = messageDateTime.secsTo(nowDateTime);
+  if (diff < 0) {
+    // The addon has a date set in the future...?
```

Comment:
Nit: this method is used for more than just addons now and the comment could get updated.

Reply:
I hear you, but I think this is fine for now, the card is needed and was tested to be useful so we can't remove it. If we want to look at the screen again later down the road we can create a ticket for it, but for now this is what was tested so it we wont make any further changes to the UI. @oskirby

Reply:
Seconding Owen's aesthetic concerns. I'll file a ticket for exploring alternatives in future iterations. 
<img width="260" alt="Screen Shot 2023-02-17 at 10 41 08 AM" src="https://user-images.githubusercontent.com/22355127/219713877-3e25faad-4bdd-4057-83e8-61e86a0d750b.png">